src/table/table.scss

```diff
@@ -16,75 +15,203 @@
     box-sizing: border-box;
   }
 
-  > .iui-tables-body {
-    width: 100%;
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
+  .iui-header {
```

Comment:
Makes sense.  Are you ok if I change `.iui-body` to `.iui-table-body` as well?  The rest of the nested classes can stay unspecific (without the `-table-` prefix).

Reply:
Make sure you double check all references of those two classes. I think there are more in the scss

Reply:
I would rename header to site-header or main-header something like that instead of renaming to iui-table-header. Header component seem like something that you put at the root of `<body>`